backend/src/main/java/com/woowacourse/gongseek/question/domain/repository/QuestionRepository.java

```diff
@@ -0,0 +1,20 @@
+package com.woowacourse.gongseek.question.domain.repository;
+
+import com.woowacourse.gongseek.question.domain.Question;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+@Repository
+@Transactional
```

Comment:
```suggestion
@Transactional
@Repository
```

중요도가 높은 어노테이션을 메서드에 가깝게 하는 것이 어떨까요?